gulpfile.js

```diff
@@ -130,6 +133,21 @@ gulp.task( 'bundle:generate',
 
 // Documentation. -------------------------------------------------------------
 
-const docsBuilder = ckeditor5DevCompiler.docs( config );
+// Documentation. -------------------------------------------------------------
+
+const ckeditor5DevDocs = require( '@ckeditor/ckeditor5-dev-docs' );
+const docsBuilder = ckeditor5DevDocs.docs( config );
+
+gulp.task( 'docs', [ 'compile:js:esnext' ], ( done ) => {
```

Comment:
Firstly we need to create a custom `runSequence` module for that. We have a ticket for this issue - https://github.com/ckeditor/ckeditor5-dev-utils/issues/21.

When the module will be ready, we can remove all sub tasks. At this moment let's leave it as it is.